maximum_grab_distance: 1.0
grab_pose_from_marker: {
  x: 0.6,
  y: 0.0,
  z: 0.0,
  qx: 0.0,
  qy: 0.0,
  qz: 0.0,
  qw: 1.0,
}
